stats/stats/src/update/mod.rs

```diff
@@ -12,6 +13,8 @@ pub trait UpdaterTrait {
         db: &DatabaseConnection,
         blockscout: &DatabaseConnection,
     ) -> Result<(), UpdateError>;
+
+    fn name(&self) -> String;
```

Comment:
I believe it would be better to return reference and convert to string only where it is required

```suggestion
    fn name(&self) -> &str;
```